doc/Doxyfile

```diff
@@ -673,7 +673,7 @@ FILE_PATTERNS          = *.c \
                          *.vhd \
                          *.vhdl
 
-# The RECURSIVE tag can be used to turn specify whether or not subdirectories 
+# The RECURSIVE tag can in turn be used to specify whether or not subdirectories
```

Comment:
We shouldn't really be modifying docs in here. It'll be fixed when we update the Doxyfile itself.